static/src/stylesheets/module/commercial/_paidfor.scss

```diff
@@ -130,6 +130,90 @@
     }
 }
 
+/* Badges
+   ========================================================================== */
+
+.commercial--paidfor {
+
+    /* Resets for new design. When (and if) the new design is committed to, the old code
+       will be torn down and these resets should no longer be necessary
+    */
+
+    .ad-slot--paid-for-badge__help {
+        display: none;
+    }
+
+    .ad-slot--paid-for-badge--front {
+        border: 0;
+        padding: 0;
+        margin: 0;
+    }
+
+    .ad-slot--paid-for-badge .ad-slot--paid-for-badge__link {
+        margin: 0;
+        float: none;
+    }
+
+    .ad-slot--paid-for-badge__link:after {
+        content: none;
+    }
+
+    .ad-slot.ad-slot--paid-for-badge--front {
+        margin: auto;
+        width: auto;
+        min-height: unset;
```

Comment:
0 seems like a safe option; everything that supports `min-height` should apply zero properly. Roger Johannson recommends it too: https://twitter.com/rogerjohansson/status/324947861148164096